src/foreign_key.c

```diff
@@ -533,6 +533,17 @@ constraint_get_trigger(Oid conoid, Oid *updtrigoid, Oid *deltrigoid)
 	table_close(rel, AccessShareLock);
 }
 
+static bool
+array_has_member(const AttrNumber *array, int len, AttrNumber member)
+{
+	for (int i = 0; i < len; i++)
+	{
+		if (array[i] == member)
+			return true;
+	}
+	return false;
```

Comment:
Weird that it's not covered, does the test work?

Reply:
oh i guess i can create some fake indexes to have it hit that path